cli/src/common/boto3/ec2.py

```diff
@@ -112,6 +112,17 @@ def describe_image_by_name_tag(self, image_name):
         owners = ["self"]
         return self.describe_images(ami_ids=[], filters=filters, owners=owners)[0]
 
+    @AWSExceptionHandler.handle_client_exception
+    def describe_instances_by_ami_id(self, image_id):
```

Comment:
method is returning just instance ids, not all the instance infos of the describe. Would "get_instance_ids_by_ami_id" fit better?

Reply:
done